tcms/testruns/templates/testruns/get.html

```diff
@@ -117,6 +117,100 @@ <h2 class="card-pf-title" style="text-align: left">
+                                    <div class="input-group">
+                                        <div class="input-group-btn">
+                                            <button type="button" class="btn btn-default dropdown-toggle" id="input-filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name <span class="caret"></span></button>
+                                            <ul class="dropdown-menu">
+                                                <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                                <li><a href="#">{% trans "Priority" %}</a></li>
+                                                <li><a href="#">{% trans "Category" %}</a></li>
+                                            </ul>
+                                        </div><!-- /input-group-btn -->
+                                        <input type="text" class="form-control" id="filter" placeholder='{% trans "Filter By Name..." %}'>
+                                    </div><!-- /input-group -->
+                                </div>
+                                <div class="form-group">
+                                    <div class="dropdown btn-group">
+                                        <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name <span class="caret"></span></button>
+                                        <ul class="dropdown-menu">
+                                            <li class="selected"><a href="#">{% trans "Name" %}</a></li>
+                                            <li><a href="#">{% trans "Priority" %}</a></li>
+                                            <li><a href="#">Last {% trans "Category" %}</a></li>
+                                        </ul>
+                                    </div>
+                                    <button class="btn btn-link" type="button">
+                                        <span class="fa fa-sort-alpha-asc"></span>
+                                    </button>
+                                </div>
+                                <div class="form-group">
+                                    <button type="button" class="btn btn-default">{% trans "Add case" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Remove case" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Update case text" %}</button>
+                                    <button type="button" class="btn btn-default">{% trans "Change assignee" %}</button>
+                                    {% comment %} TODO: dropdown or modal - https://www.patternfly.org/v3/pattern-library/widgets/#modal {% endcomment %}
+                                    <button type="button" class="btn btn-default">{% trans "Change status" %}</button>
```

Comment:
Statuses are coming from the DB and should be a drop-down action. Similar thing exists for the TestPlan page.

Reply:
still unresolved

Reply:
^^^